appshell/mac/Brackets.sh

```diff
@@ -0,0 +1,8 @@
+#!/bin/bash
+
+# Give preference to launching the one in /Applications folder
```

Comment:
It seems like the opposite would be more useful -- if the user has manually configured their PATH to point to another copy of Brackets (e.g. a dev copy), that should take precedence over the default command-line location.